### DISCUSSION TITLE
Welcome to pyHYBRO Discussions!

### DISCUSSION THREAD

**Discussion by paulflang** [HYBRO-dev/HYBRO#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paulflang** [comment]
Hi,
Just trying this (new?) discussion feature. Could be a better communication channel than email. What do you think?

**Comment by paulstapor** [comment]
Nice, thanks! Yes, seems fine to me.
